src/CMakeLists.txt

```diff
@@ -17,10 +17,11 @@ message (STATUS "CMake build configuration for ${PROJECT_NAME} (${CMAKE_BUILD_TY
 add_library(musica)
 add_library(musica::musica ALIAS musica)
 
-if (NOT ${MUSICA_GPU_TYPE} STREQUAL "None")
+if (NOT ${MUSICA_GPU_TYPE} STREQUAL "None" AND NOT APPLE)
```

Comment:
Just want to leave a minor comment here that we only want to enable the `MUSICA_ENABLE_CUDA` option when there is an NVIDIA GPU provided. In the very further future when we want to test the AMD GPU, we may need a similar option like `MUSICA_ENABLE_HIP` instead.